INPUT_OUTPUT_CHANNEL_BIT_DESCRIPTIONS.s

```diff
@@ -72,7 +72,7 @@
 
 # CHANNEL 12	CHAN12: OUTPUT CHANNEL; BITS USED TO DRIVE NAVIGATION AND SPACECRAFT HARDWARE.
 #
-#		BIT 1		ZERO RR CDU; CDU'S GIVE RRADAR INFORMATION FOR LM
+#		BIT 1		ZERO RR CDU; CDU'S GIVE RADAR INFORMATION FOR LM
```

Comment:
`RRADAR` is used in [source](http://www.ibiblio.org/apollo/ScansForConversion/Luminary099/0055.jpg)